### DISCUSSION TITLE
Samsung

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2924] [Show and tell]
There are reports that applying a Samsung Android update released in June has resulted in intermittent connectivity.
If you have a Samsung phone, and use xDrip as a collector from G5,G6, Dexcom One, G7, or Libre, keep that in mind.

If you are using xDrip as a follower, meaning something else is the collector, you can ignore this post. CareLink, Companion app, Web follower, xDrip sync, Nightscout follower, or BYODA are some examples.

**Comment by AlisaAqua** [comment]
Hy!
I'm new here and unfortunately a complete newbie with this site and xdrip app.
I have Samsung s10+ and I have tried to install dozen of times xdrip+ but it just won't install itself. I've tried latest official version which was in May, I also tried varanty version 4, and I've also tried latest prerelease version which was released few days ago. Every time I get same result.
First my phone tells me that this is app is made for older phone ver but I still click install, the installation is running and then at the end it says that app couldn't be installed.
I guess from your post it's related to update you've mentioned, right?
I truly hope you guys will be able to fix this. In my country we just got libre2 so until now I didn't need xdrip because the alarms in ver1 weren't available so I only used librelink. Unfortunately i can't undo the update of IOS 😪
So I really hope you'll be able to make the app available for Samsung phones. Thank you for all your hard work. 
Best regards ✨️

**Reply by Navid200** [reply]
Please open a new thread and I will help you.
You have posted in a thread about a very specific topic.  What you are asking for needs it own thread.
Go to the page linked below and click on "New discussions" in green at top right.
https://github.com/NightscoutFoundation/xDrip/discussions

**Comment by aleks372** [comment]
Is there any list of confirmed Samsung devices being affected?

S23 Ultra with June update + xDrip 06.29 works flawlessly for 3 weeks straight.
Also other comments in FB post mention that other S series devices also work fine, maybe just A series problem?

**Reply by Navid200** [reply]
> S23 Ultra with June update + xDrip 06.29 works flawlessly for 3 weeks straight. 

What is the hardware data source set to?
What Samsung patch is on the phone?

**Reply by aleks372** [reply]
Data source is Dex1 Transmitter
Security Patch 1 June 2023,

**Reply by aleks372** [reply]
@Navid200 sure, but what SMR under patch stands for? firmware region or just like Security Patch level date?

**Comment by bart2** [comment]
That post got me thinking... Should xDrip support sending important messages from developers to its users within the app? If I wasn't involved in a bit of xDrip development right now, I would definitely miss this message.

If a security update is likely to cause issues with xDrip operations, we should probably make an effort to make users aware.

Maybe some kind of push notification with a short message and a link to a page with more information/status updates?

**Comment by swissalpine** [comment]
I have the June update on my Samsung Galaxy Z Flip 4 and am very surprised by the discussion: my G6 capture rate is still between 99 and 100%.

**Reply by Navid200** [reply]
I would like to create a table listing models that have the problem and those that don't.

**Reply by bart2** [reply]
Would it be possible to add diagnostic information gathering into the app itself?

We could ask users if they agree to send anonymized diagnostic information and then analyze which phone models start hitting issues, without users having to manually report this stuff to the developers.

BTW, Samsung Galaxy S22+ with a June update and no issues with Dexcom G6 capture rate.

**Reply by Navid200** [reply]
Is this what you mean?
https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html

**Comment by Navid200** [comment]
| User | Model | Working | 
| ----- | ------- | -------------- |  
| @swissalpine | Z flip 4 | Works fine |
